backend/transcribee_backend/routers/document.py

```diff
@@ -57,6 +69,7 @@ def create_default_tasks_for_document(session: Session, document: Document):
         task_type=TaskType.TRANSCRIBE,
         task_parameters={"lang": "auto", "model": "base"},
         document_id=document.id,
+        dependencies=[reencode_task],
```

Comment:
is this needed?

Reply:
Yes, `load_audio` only supports a small number of formats (eg doesn’t do mp4) -> ffmpeg converts them to a safe format

Reply:
Hm... can we skip the dependency if we are likely to be able to load it directly? Or use a library that can load more formats? This seems like a downgrade performance-wise :/

Reply:
I'd prefer not to, as it also calms my mind a bit to know that (after it went through ffmpeg, which is it list fuzzed a bit), the files are in a sortof-safe state. Also the reencoding is incredibly fast